english/new-movie/new-movie.md

```diff
@@ -0,0 +1,27 @@
+Scratched on the wall you read...
+
+> No man is an island,
+> Entire of itself.
+> Each is a piece of the continent,
+> A part of the main.
+
+Further down this abandoned station the following catches your eye...
+
+> Be still.
+> Listen to the stones of the wall.
+
+It *is* eerie quiet down here, except for a lone water drip and a machine that clanks every minute in the distance.
+
+But then the following freezes you.
+
+> I’m still turning my stick.
+> He’s already eaten six.
+
+More of this [marshmellow madness](http://www.poetryfoundation.org/poem/241898) every where you go! 
+
+But you got somewhere to be, friends to meet, a movie to catch, and this shortcut no longer feels like a good idea.
+
+The marshmellow mystery is going to have to wait. WHO has eaten six? Is there something significant about the number? Clues, clues, everywhere clues that don't connect!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.